packages/project-build/src/schema/style-source-selections.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from "zod";
+
+const InstanceId = z.string();
```

Comment:
Shouldn't we use it from the original place where we have instance id?

Reply:
I'd prefer to not export these values separately and Instance.id thing not possible. So let's keep this way. It's more like documentation than any functionality.